apps/condo/domains/meter/access/MeterReportingPeriod.js

```diff
@@ -42,44 +49,61 @@ async function canReadMeterReportingPeriods ({ authentication: { item: user } })
     }
 }
 
-async function canManageMeterReportingPeriods ({ authentication: { item: user }, originalInput, operation, itemId }) {
+async function canManageMeterReportingPeriods ({ authentication: { item: user }, originalInput, operation, itemId, itemIds }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isSupport || user.isAdmin) return true
 
-    if (operation === 'create') {
-        const orgId = get(originalInput, ['organization', 'connect', 'id'], null)
-        const propertyId = get(originalInput, ['property', 'connect', 'id'], null)
-        let propertyOrganization
-
-        if (propertyId) {
-            const property = await getByCondition('Property', {
-                id: propertyId,
-                deletedAt: null,
-            })
+    const isBulkRequest = Array.isArray(originalInput)
 
-            if (!property) return false
+    let organizationIds = []
 
-            propertyOrganization = get(property, 'organization', null)
+    if (operation === 'create') {
+        if (isBulkRequest) {
+            return false
+        } else {
```

Comment:
Why you can't use bulk create, but can use bulk update?

Reply:
I don't think there is a need for this right now